RTech.CoffeeMachine.Api/BrewCoffee/Controllers/CoffeeMachineController.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Caching.Memory;
+using RTech.CoffeeMachine.Api.BrewCoffee.Models;
+using RTech.CoffeeMachine.Api.BrewCoffee.Util;
+
+namespace RTech.CoffeeMachine.Api.BrewCoffee.Controllers;
+
+[ApiController]
+[Route("")]
+public class CoffeeMachineController : ControllerBase
+{
+    private readonly ILogger<CoffeeMachineController> _logger;
+    private readonly IDateTimeProvider _dateTimeProvider;
+    private readonly IBrewStatusProvider _brewStatusProvider;
+    // Interviewer note: MemoryCache from Microsoft.Extensions.Caching.Abstractions is thread safe
+    private readonly IMemoryCache _cache;
+    public static readonly string CACHE_KEY_NUM_REQUESTS = $"{nameof(CoffeeMachineController)}:numRequests";
```

Comment:
Since this variable is referenced is another separate assembly, im straying away from `const` & instead prefer to use `static`